tasks/utilities/data_handling/task_fastq_handeling.wdl

```diff
@@ -0,0 +1,47 @@
+version 1.0
+
+task sra_lite_autodetect {
```

Comment:
can you rename this file so that fastq_handling is spelt correctly? 😂

Reply:
Oh my god! 😬